File: dataset/output/MISTRAL_codestral-latest/task95/Task95_MISTRAL_codestral-latest.java
// Java does not have a built-in function to get the processor time in seconds.
// However, you can use the following code to get the CPU time used by the current thread in nanoseconds.

long startTime = System.nanoTime();
// Your code here
long endTime = System.nanoTime();
long duration = (endTime - startTime) / 1000000000;  // divide by 1000000000 to get seconds
System.out.println("Processor time: " + duration + " seconds");

File: dataset/output/MISTRAL_codestral-latest/Task95_MISTRAL_codestral-latest.cpp
#include <iostream>
#include <ctime>

int main() {
    clock_t start = clock();
    // Your code here
    clock_t end = clock();
    double duration = (double)(end - start) / CLOCKS_PER_SEC;
    std::cout << "Processor time: " << duration << " seconds" << std::endl;
    return 0;
}